common/sagas/schedule/paramsValidity.ts

```diff
@@ -0,0 +1,154 @@
+
+    const isOffline: boolean = yield select(getOffline);
+    const schedulingToggle: ICurrentSchedulingToggle = yield select(getSchedulingToggle);
+    const scheduling = Boolean(schedulingToggle && schedulingToggle.value);
+
+    if (isOffline || !scheduling) {
+      continue;
+    }
+
+    yield call(checkSchedulingParametersValidity);
+  }
+}
+
+function* checkSchedulingParametersValidity() {
+  const currentTo = yield select(getCurrentTo);
+  const currentValue = yield select(getCurrentValue);
+  const callData = yield select(getData);
+  const scheduleType: ICurrentScheduleType = yield select(getScheduleType);
+  const windowStart: ICurrentWindowStart = yield select(getWindowStart);
+  const windowSize: ICurrentWindowSize = yield select(getWindowSize);
+  const timeBounty: ICurrentTimeBounty = yield select(getTimeBounty);
+  const scheduleGasPrice = yield select(getScheduleGasPrice);
+  const scheduleGasLimit = yield select(getScheduleGasLimit);
+  const deposit = yield select(getScheduleDeposit);
+  const node = yield select(getNodeLib);
+  const wallet = yield select(getWalletInst);
+  const scheduleTimestamp: ICurrentScheduleTimestamp = yield select(getScheduleTimestamp);
+  const scheduleTimezone: ICurrentScheduleTimezone = yield select(getScheduleTimezone);
+
+  if (
+    !currentValue.value ||
+    !currentTo.value ||
+    !scheduleGasPrice.value ||
+    !wallet ||
+    !windowSize.value ||
+    !(windowStart.value || scheduleTimestamp.value)
+  ) {
+    return;
+  }
+
+  const callGasLimit = scheduleGasLimit.value || EAC_SCHEDULING_CONFIG.SCHEDULE_GAS_LIMIT_FALLBACK;
+
+  const endowment = calcEACEndowment(
+    callGasLimit,
+    currentValue.value || new BN(0),
+    scheduleGasPrice.value || gasPriceToBase(EAC_SCHEDULING_CONFIG.SCHEDULE_GAS_PRICE_FALLBACK),
+    timeBounty.value
+  );
+
+  const fromAddress = yield apply(wallet, wallet.getAddressString);
+
+  const data = getValidateRequestParamsData(
+    bufferToHex(currentTo.value),
+    callData.value ? bufferToHex(callData.value) : '',
+    callGasLimit,
+    currentValue.value,
+    windowSizeBlockToMin(windowSize.value, scheduleType.value),
+    calculateWindowStart(
+      scheduleType.value,
+      scheduleTimestamp,
+      scheduleTimezone.value,
+      windowStart.value
+    ),
+    scheduleGasPrice.value,
+    timeBounty.value,
+    deposit.value || new BN(0),
+    scheduleType.value === 'time',
+    endowment,
+    fromAddress
+  );
+
+  const callResult: string = yield apply(node, node.sendCallRequest, [
+    {
+      to: EAC_ADDRESSES.KOVAN.requestFactory,
+      data
+    }
+  ]);
+
+  const { paramsValidity } = RequestFactory.validateRequestParams.decodeOutput(callResult);
+
+  const errors = parseSchedulingParametersValidity(paramsValidity);
+  const paramsValid = errors.length === 0;
+
+  yield call(setField, {
+    raw: paramsValid,
+    value: paramsValid
+  });
+}
+
+export function* setField(payload: SetScheduleParamsValidityAction['payload']) {
```

Comment:
Same here as above comment ^ (concerning just inlining `yield put(setScheduleParamsValidity(payload));` to avoid indirection)